{
    "main.debug.service_unfinish": "在等待服务完成时出错。",
    "main.success.start_service_done": "服务启动成功。耗时：${time} 秒。",
    "main.success.service_exit": "已成功退出服务。",
    "cluster.success.load_cluster": "已成功加载节点 [${cluster}]。",
    "cluster.debug.base_url": "使用的地址是 [${base_url}]",
    "cluster.success.fetch_filelist": "已成功获取文件列表，数量 [${total}] 大小 [${size}]",
    "cluster.debug.fetch_token": "正在获取 Token (${cluster})",
    "cluster.debug.fetch_token_success": "已成功获取 Token (${cluster})，有效期为 [${ttl}]",
    "cluster.error.unspported_storage": "不支持的存储类型：[${type}] [${path}]",
    "cluster.success.load_storage": "已成功加载 [${type}] 路径为 [${path}]",
    "cluster.info.sync_configuration": "同步策略来源于 [${source}] 并发速度为 [${concurrency}]",
    "cluster.processbar.download_files": "下载文件",
    "i18n.prompt.failed": " 未找到 i18n 字符串",
    "cluster.warning.no_check_function": "没有找到检查函数，使用类型为 exists",
    "clusters.error.downloading": "下载时候出错，原因 [${type}]，文件 [${file_path} ${file_hash}(${file_size})] 下载后的哈希值 [${hash}]，主机 [${host}] 下载时候的历史记录：${responses}",
    "web.debug.local_port": "内部开放端口为 [${port}]",
    "web.debug.ssl_port": "内部 SSL 开放端口为 [${port}]",
    "web.success.public_port": "当前 Web 服务器在 [${port}] 端口上开启",
    "web.error.any": "Web 服务器出错 ${e}",
    "web.debug.request_info": "${host} | ${time} | ${address} | ${method} ${status} | ${real_path} - ${user_agent}",
    "cluster.error.socketio.request_cert": "节点 [${cluster}] 请求证书失败，原因 [${err}]",
    "cluster.debug.public_host": "已在 [${host}:${port}] 上开放",
    "cluster.error.socketio": "节点 [${cluster}] 发送 [${type}] 时出错，原因 [${err}]",
    "cluster.info.socketio.message": "节点 [${cluster}] 主控消息：${message}",
    "storage.error.alist": "Alist 返回出错，错误代码 [${status}]，原因 [${message}]",
    "storage.error.alist.fetch_token": "Alist 获取 Token 出错，错误代码 [${status}]，原因 [${message}]",
    "cluster.success.keepalive": "节点 [${cluster}] 保活成功，已回原 [${hits}] 个文件，共计 [${bytes}] 流量，延迟为 [${ping}ms]",
    "cluster.debug.socketio.disconnected": "SocketIO 已断开连接",
    "cluster.debug.socketio.connected": "SocketIO 已连接",
    "cluster.success.cluster.disable": "节点 [${cluster}] 已正常下线",
    "cluster.info.want_enable": "节点 [${cluster}] 正在启用中……",
    "cluster.info.cluster.retry_enable": "节点 [${cluster}] 将在 [${delay}] 后重新启用",
    "cluster.success.enabled": "节点 [${cluster}] 已正常上线",
    "cluster.error.socketio.message": "节点 [${cluster}] 收到错误消息，原因 [${message}]",
    "cluster.error.socketio.warden": "[Warden] 节点 [${cluster}] 收到错误消息，原因 [${message}]",
    "utils.error.service_error": "服务出错，状态码 [${code}(${httpCode})] 类型 [${name}] 信息 [${message}]"
}